code/components/jomjol_image_proc/CImageBasis.cpp

```diff
@@ -608,7 +634,7 @@ CImageBasis::~CImageBasis()
     RGBImageLock();
 
     if (!externalImage)
-        stbi_image_free(rgb_image);
+        free_psram_heap(std::string(TAG) + "->rgb_image (~CImageBasis)", rgb_image);
```

Comment:
Have you checked if in the function stbi_image_free something is handled special that you can repalce it?
Comment valid for all stbi_image_free usages